core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
Ah, right. Didn't think of that. Then let's see what you come up with in the Huddle. I am +1 for using Optionals in the future and creating an issue that we should tackle that in version ``2.0``.